CHANGES_NEXT_RELEASE

```diff
@@ -7,3 +7,6 @@
        a consequence, the 'attributeFormat' URI parameter is no longer needed in the 'GET /v1/contextTypes', so it is ignored.
 - Fix: DELETE updates are resolved locally, previously they were forwarded to CPrs (Issue #755)
 - Fix: The URI parameter 'notifyFormat' is now treated the same way for subscriptions as for registrations.
+- Add: Notifications and forwarded queries/updates are now controlled by a timeout. (Issue #880)
+       The default timeout is set to 5000 milliseconds (5 seconds), including connection, writing the request and reading the response.
+       To change this timeout, a new CLI option '-appTmo' has been added to the broker.
```

Comment:
ok to modify the parameter names.
I used 'app' as a concept that covers both notifications and forwards.
We use 'application' in the presentations.
Only 'notify' isn't good enough as it implies that the timeout is only for notifications.
We need something that covers both notifications and forwards to updates and queries ...